earthfile2llb/earthfile2llb.go

```diff
@@ -247,12 +247,19 @@ func Earthfile2LLB(ctx context.Context, target domain.Target, opt ConvertOpt, in
 		opt.MainTargetDetailsFuture = nil
 	}
 	if found {
+		// The found target may have been initially referenced by a FROM or a COPY
+		// if it is referenced a second time by a BUILD, it may contain items that
+		// require a save (export to the local host) or a push
```

Comment:
Is this two sentences or one? `The found target may have been initially referenced by a FROM or a COPY if it is referenced a second time by a BUILD` reads as if it should be two sentences, I think, but I'm not 100% sure.

Reply:
`; however, ` to the rescue :)